cli/src/test-utils/test-logger.ts

```diff
@@ -18,7 +18,7 @@ function log(message: string) {
 }
```

Comment:
I'm gonna comment on this `chalk.white` why `white`? Why can't it be the default colour on the console's theme itself?